spyder/plugins/editor/widgets/recover.py

```diff
@@ -147,8 +149,8 @@ def add_table(self):
         table.setHorizontalHeaderLabels(labels)
         table.verticalHeader().hide()
 
-        table.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
-        table.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
+        table.setHorizontalScrollBarPolicy(Qt.ScrollBarAsNeeded)
```

Comment:
Should not this be off? by default?

Reply:
Maybe if I set the width using the table width I can disable this, should I do it?

Reply:
It is ok, we can merge as is!

Reply:
> Maybe if I set the width using the table width I can disable this, should I do it?

The dialog can get arbitrarily wide if you have really long directory or file names, so I think it is dangerous to use the table width.